From 57443666b05a056f856e18407919de184bb69afd Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Tue, 3 Apr 2018 17:11:15 +0200
Subject: [PATCH 03/11] force root be rw before localmount

The service that pulls in root remount is mtab which we dont need/use.
---
 init.d/localmount.in | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/init.d/localmount.in b/init.d/localmount.in
index 6f57f681..14189396 100644
--- a/init.d/localmount.in
+++ b/init.d/localmount.in
@@ -13,9 +13,9 @@ description="Mounts disks and swap according to /etc/fstab."
 
 depend()
 {
-	need fsck
-	use lvm modules root
-	after clock lvm modules root
+	need fsck root
+	use lvm modules
+	after clock lvm modules
 	keyword -docker -jail -lxc -prefix -systemd-nspawn -vserver
 }
 
-- 
2.33.0

